nx-X11/programs/Xserver/hw/nxagent/Handlers.c

```diff
@@ -963,12 +963,16 @@ void nxagentHandleCollectInputFocusEvent(int resource)
    * Calling functions from nxcompext from nxagent sounds like a very bad idea
    * to begin with, but let's assume that's necessary for now and work around
    * the corruption issue.
+   *
+   * What we're doing here will lead to compile warnings when XlibWindow is
+   * actually a Window64, but that's "really" the type of "Window" libcompext
+   * expects.
+   *
+   * One way to maybe work around this issue is to define Window, XID and maybe
+   * Atom to their Xlib* types, include NXlib.h and then undefine these macros
+   * again. Might have unforseen consequences, though...
    */
```

Comment:
I've replaced the macro-condition with XlibWindow now. Please take a look at the updated comment, though. Is doing the workaround hack a viable way to silence the compile warnings on 64bit platforms? I really don't know what other consequences this might have, but generally libcompext is Xlib-based so all types in there should be "supposed" to be Xlib types.